README.md

```diff
@@ -1 +1,367 @@
 # Scoobie
+
+[![Buildkite pipeline](https://badge.buildkite.com/d93da6efa412a5c2bb112e92443a558c2d291eafa2d8807c00.svg?branch=master)](https://buildkite.com/seek/scoobie)
+[![npm package](https://img.shields.io/badge/npm-%40seek%2Fscoobie-cc3534.svg)](https://www.npmjs.com/package/@seek/scoobie)
+[![Slack channel](https://img.shields.io/badge/slack-%23indirect--apply-3f0f3f.svg)](https://slack.com/app_redirect?team=T02P37LGR&channel=indirect-apply)
+[![GitHub repos](https://img.shields.io/badge/users-@SEEK--Jobs-0d3880.svg)](https://github.com/search?q=filename%3Apackage.json+org%3ASEEK-Jobs+-repo%3ASEEK-Jobs%2Fzactive-directory+%22%40seek%2Fscoobie%22&type=Code)
+[![Upkeep](https://img.shields.io/badge/users-Upkeep-61d3b1.svg)](https://upkeep.ssod.skinfra.xyz/package#?name=@seek/scoobie)
```

Comment:
Is it intentional to leave the ssod and zactive references?

Reply:
I think this was wrong in the original. Will fix.